templates/layout.html

```diff
@@ -70,9 +70,9 @@
                               ('Praha – Datový kurz',
                                   'https://naucse.python.cz/2020/pydata-praha-jaro/'),
                               ('Ostrava – začátečnický kurz',
-                                  'https://naucse.python.cz/2020/pyladies-ostrava-jaro/'),
-                              ('Ostrava – kurz datové analýzy',
-                                  'https://naucse.python.cz/2020/pydata-ostrava-jaro/'),
```

Comment:
Fakt měly pydata zmizet?

Reply:
Datový kurz už proběhl a další v plánu zatím není, takže je to dle mého ok.